examples/run_MWE.py

```diff
@@ -4,141 +4,129 @@
 import socket
 import sys
 import time
+from Instruments.XEPR_eth import xepr
+
 fl = figlist_var()
-#{{{ Verify arguments compatible with board
-def verifyParams():
```

Comment:
ask alec what this is?

Reply:
"modified working example" "can safely ignore it"